2022/12/v3-8-0-released/index.html

```diff
@@ -729,27 +727,28 @@ <h3>Workflow engine graceful shutdown</h3>
 						<p>
 							StackStorm’s Workflow engine can now handle shutdown events more gracefully thanks to the
 							contribution by <a href="https://github.com/khushboobhatia01" target="_blank">@khushboobhatia01</a>
-							from VMware.
+							from VMware in <a href="https://github.com/StackStorm/st2/pull/5463/" target="_blank">PR#5463</a>
 						</p>
 						<p>
 							Two new configurations has been added to <code>st2.conf</code> in the <code>[workflow_engine]</code>
 							section to have more granular control:
 						</p>
 						<p>
-							<ul>
-								<li><code>exit_still_active_check = 300</code><br>
-									How long to wait for process (in seconds) to exit after receiving shutdown signal.
-								</li>
-								<li><code>still_active_check_interval = 2</code><br>
-									How long to wait for process (in seconds) to exit after receiving shutdown signal.
-								</li>
-							</ul>
+						<ul>
+							<li><code>exit_still_active_check = 300</code><br>
+								How long to wait for process (in seconds) to exit after receiving shutdown signal.
+							</li>
+							<li><code>still_active_check_interval = 2</code><br>
+								Time interval between subsequent queries to check executions handled by WFE.
```

Comment:
```suggestion
								Time interval between subsequent queries to check executions handled by the Workflow Engine.
```

Let's avoid abbreviations as they're usually confusing for the user.